### DISCUSSION TITLE
Is there any difference between *-amd64-v3.zip and *-amd64.zip?

### DISCUSSION THREAD

**Discussion by pedoc** [xjasonlyu/tun2socks#263] [Q&A] [answered]
The release list contains some variants of v3, what does this mean?

**Comment by xjasonlyu** [comment, marked as answer]
They are GOAMD64 variables. v3 version supports some newer x86 instructions that can be executed faster but requires newer x86 chips; non-v3 version is a generic x86 binary that runs on all x86 platforms.

**Reply by pedoc** [reply]
thanks
